lib/signs_ui/signs/messages.ex

```diff
@@ -97,7 +97,15 @@ defmodule SignsUi.Signs.Messages do
 
       {zone, line_no} = String.split_at(zone_line, 1)
       line_no = String.to_integer(line_no)
-      {duration, zone, line_no, List.first(text_list)}
+
+      display_text =
+        case text_list do
+          [_stopped, n_stops, _away] -> n_stops
```

Comment:
I think the message comes in as `[n_stops, _away, _stopped]`.

Reply:
oh, ok.  is that because of sorting alphabetically? i  see the to_string at least had the order as stopped n_stops away

```
 63     def to_string(%{headsign: headsign, stops_away: n}) do
 64       stop_word = if n == 1, do: "stop", else: "stops"
 65
 66       pages = [
 67         Content.Utilities.width_padded_string(headsign, "Stopped", 18),
 68         Content.Utilities.width_padded_string(headsign, "#{n} #{stop_word}", 18),
 69         Content.Utilities.width_padded_string(headsign, "away", 18)
 70       ]
 71
 72       {pages, 3}
 73     end
```
which is what i went based on

Reply:
oh wait the to string is the signs display so i guess thats obviously how this would be.

Reply:
Yeah it comes in out of order because of a bug in the actual ARINC code which we of course have to replicate lol